tensorflow/compiler/tf2tensorrt/kernels/trt_engine_op.cc

```diff
@@ -505,6 +502,12 @@ TRTEngineOp::TRTEngineOp(OpKernelConstruction* context)
       [](PartialTensorShape shape) { return !shape.IsFullyDefined(); });
   VLOG(2) << "TRTEngineOp has_dynamic_shape_input_: "
           << has_dynamic_shape_input_;
+
+  if (has_dynamic_shape_input_ && !use_implicit_batch_) {
+    OP_REQUIRES(context, !calibration_mode_,
+                errors::InvalidArgument(
+                    "Dynamic shape mode does not support calibration"));
+  }
```

Comment:
This change doesn't seem to match the description of this PR "Re-enable INT8 calibration with implicit batch mode".
IIUC,  the PR description should be "Enable INT8 calibration for use_implicit_batch=false when there is no dynamic shape inputs".
And the detail description of the PR is "Previously we disable INT8 calibration when use_implicit_batch=false. This PR partially enable INT8 calibration for use_implicit_batch=false, that is, when the engine has only static input shapes."